mantidproject/mantid
# Require python2-matplotlib >= 1.5.2 on RHEL 7

Places a minimum requirement of matplotlib >= 1.5.2 on the RHEL 7 rpm. The rpms were built on the [mantid copr](https://copr.fedorainfracloud.org/coprs/mantid/mantid/) and are also in the ISIS repository.

**To test:**

Build a package on RHEL 7 and verify it requires `python2-matplotlib >= 1.5.2`

Fixes #16873

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
